app/src/main/java/com/github/se/eventradar/ui/messages/Messages.kt

```diff
@@ -225,10 +227,18 @@ fun MessagePreviewItem(
       verticalAlignment = Alignment.CenterVertically) {
         Image(
             painter =
-                painterResource(
-                    id = R.drawable.placeholder), // TODO: replace with actual profile pic
+                rememberAsyncImagePainter(
+                    ImageRequest.Builder(LocalContext.current)
```

Comment:
Question: Was there a reason you used rememberAsyncImagePainter, instead of using AsyncImage() directly?

Reply:
Nope, if I'm being completely honest Copilot did this for me. I did some research and `AsyncImage` is a little more reliable so I will change it.